<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PictureTipsWidget</class>
 <widget class="QWidget" name="PictureTipsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#PictureTipsWidget{background-color:white;}</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>20</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>步骤二</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>步骤一</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="step2_GroupBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>590</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#step2_GroupBox{background: #E4F4FC;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="step2Picture_Label">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>60</y>
      <width>386</width>
      <height>280</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="step1_GroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>590</width>
     <height>400</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#step1_GroupBox{background: #E4F4FC;
border-radius: 12px;}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="step1Picture_Label">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>0</y>
      <width>376</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="tip1_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>520</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tip2_Label">
   <property name="geometry">
    <rect>
     <x>712</x>
     <y>520</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
